docs/sources/developer/architecture/components/unpatched-console.md

```diff
@@ -0,0 +1,8 @@
+# Unpatched Console
+
+The unpatched console refers to an unmodified `console` object that is used for various purposes like defining the
+internal Faro logger (check below) or pointing out an alternative to the `console` object for the apps that are using
+a custom logger instead.
```

Comment:
(check below) -> (see below)?
"or pointing out an alternative" -> "or pointing *to* an alternative"? (not sure on this one)